### PR TITLE
Westmidlands Class 2 - Senait Ekubai -Github & Github Desktop- WEEK_1

### PR DESCRIPTION
**Pull request by SenaitEkubai**
# Your Details

- Your Name: Senait Ekubai
- Your City:Birmingham
- Your Slack Name:Senait Ekubai

# Homework Details

- Module:Github and Github Desktop
- Week:1

### CODE REVIEW

**Review by Know-Thyself — APPROVED** [review]
I believe that you have done a pretty good job.
Well done Senait!

**Comment by mikinet** [comment]
Good product you've got there. I figure you must have forgotten to fill the details in the file by accident. You can make your assignment complete by doing so.

**Review by mikinet — CHANGES_REQUESTED** [review]
I see most of your answers are better than mine, Senait. It'll be perfect if you go through it once more and eliminate the few discrepancies. Cheers.
Also, feel free to review mine, if you've got the time. I'd appreciate it. Thanks!

File: homework.md [hunk 1]
```diff
@@ -4,53 +4,51 @@
...
 
-Hyper Tall Massive Language
+Hyper Text mark-up Language 
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Cascading Style Sheets
```
Comment (ironkamosi) [inline homework.md:31]: Hi Senait,

Your answer explains what CSS stands for. Can you briefly explain how and why it is used ?

All the best,

Aaron

**Pull request closed without merging**